### ISSUE TITLE
Create CSS variables from _variables.scss values

### ISSUE THREAD

**Issue by helgatheviking** [woocommerce/woocommerce#28707]
**Is your feature request related to a problem? Please describe.**
It is sometimes useful to design an element in a plugin/theme and use the same color scheme as WooCommerce core. Currently you need to include the `_variables.scss` mix in and use it to compile a CSS file.

**Describe the solution you'd like**
What I think could work well (and not really change anything for existing users) would be to define some CSS variables like so:

```
:root {
	--woocommerce:   	$woocommerce;
	--wc-green:         $green;
	--wc-red:           $red;
	--wc-orange:        $orange;
	--wc-blue:          $blue;
	--wc-primary:       $primary;                                    
	--wc-primarytext:   $primarytext;
	--wc-secondary:     $secondary
	--wc-secondarytext: $secondarytext;
	--wc-highlight:     $highlight;
	--wc-highlightext:  $highlightext;
	--wc-contentbg:     $contentbg;                                       
	--wc-subtext:       $subtext;
}
```

**Comment by kevinruscoe** [comment]
Terrific idea, no idea why this hasn't been mentioned before!

**Comment by helgatheviking** [comment]
Thanks for creating the PR, @kevinruscoe !

**Comment by ninetyninew** [comment]
@helgatheviking @kevinruscoe Just spotted this recently when I noticed the CSS variables in the debugger. Could we use these variables for referencing colors in the Woo stylesheets?

I suspect then if I needed to change everywhere WooCommerce uses red to be a website's red color from their brand guidelines then rather than adding rules for each individual element and maintaining that I can just override the root variables? I've not used pure CSS variables before (only SCSS) but I assume that's how they work?

I'm seeing the SCSS files like this one hardcode a color:
https://github.com/woocommerce/woocommerce/blob/23710744c01ded649d6a94a4eaea8745e543159f/assets/css/woocommerce.scss#L85

I raised this ticket a while ago https://github.com/woocommerce/woocommerce/issues/28546 which is related.
